packages/ui/src/Popup/AuthManagement/WebsiteEntry.tsx

```diff
@@ -19,15 +21,21 @@ function WebsiteEntry ({ className = '', info, toggleAuth, url }: Props): React.
 
   return (
     <div className={`${className} ${info.isAllowed ? 'allowed' : 'denied'}`}>
```

Comment:
Instead of a `div` here, it could be wrapped in a Box/Flex with left and right padding `<Box px='s'`. This will allow matching the designs better with the separation lines spanning all the way from left to right. I added a related comment on `AuthManagement/index.tsx` about it, since the parent containers padding has to be modified as well